tests/flow/test_config.py

```
@@ -100,6 +100,13 @@ def test03_config_set(self):
         expected_response = config_value
         self.env.assertEqual(response, expected_response)
 
+    def test03_config_set_async_delete_rejected(self):
+        try:
+            self.db.config_set("ASYNC_DELETE", 1)
+            assert(False)
```

Ruff (0.15.14)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)